docs/SETUP_CI_CD_PIPELINE.md

```diff
@@ -161,34 +249,114 @@ They have two very important use:
+>So if you need to deploy a live environment, you will need to request a quota increase for the region you are deploying into. 
+>Ask a core team member to do it for you.
+
+##### AWS credentials
+
+The credentials obtained from the AWS dashboard needs to be stored locally at the path `~/.aws/credentials` under a profile called `[gigadb]`
+
+if the file doesn't exist yet, it should looks something like:
+
+```
+[gigadb]
+aws_access_key_id=XXXXX
+aws_secret_access_key=YYYYY
+```
+
 #### GitLab Variables
 
-Ensure the following variables are set for their respective environments in the appropriate GitLab project (the variables should have the Protected checkbox unchecked if it's your personal project space in the Forks group)
+Ensure the following variables are set for their respective environments in the appropriate GitLab project.
+Make sure the "Protect variable" and "Expand variable reference" checkboxes are unchecked.
+the Visibility radio input should be set to "Visible" except for the passwords and tokens that should be set to "Masked".
 
-| Name | Masked? |
+| Name | value | 
 | --- | --- |
-| DEPLOYMENT_ENV | no |
-| REMOTE_HOME_URL | no |
-| REMOTE_HOSTNAME | no |
-| REMOTE_PUBLIC_HTTP_PORT | no |
-| REMOTE_PUBLIC_HTTPS_PORT | no |
-| REMOTE_SMTP_HOST | no |
-| REMOTE_SMTP_PASSWORD | yes |
-| REMOTE_SMTP_PORT | no |
-| REMOTE_SMTP_USERNAME | no |
-| GITLAB_PRIVATE_TOKEN | yes |
-| gigadb_db_host | no |
-| gigadb_db_user | no |
-| gigadb_db_password | yes |
-| gigadb_db_database | no |
-| fuw_db_host | no |
-| fuw_db_user | no |
-| fuw_db_password | yes |
-| fuw_db_database | no |
-| PORTAINER_PASSWORD | yes |
-
-
-so, there should be 2 or 3 versions of each variable, one for each environment.
+| DEPLOYMENT_ENV | deployment environment goes here |
+| REMOTE_HOME_URL | URL to the home website as https://FQDN |
+| REMOTE_HOSTNAME | domain name associated to the elastic IP of the web server as FQDN |
+| REMOTE_PUBLIC_HTTP_PORT | 80 |
+| REMOTE_PUBLIC_HTTPS_PORT | 443 |
+| REMOTE_SMTP_HOST | Pick an SMTP host |
+| REMOTE_SMTP_PASSWORD | SMTP password |
+| REMOTE_SMTP_PORT | 563 |
+| REMOTE_SMTP_USERNAME | SMTP username |
+| gigadb_db_host | keep empty, it will be overwritten by the provising script with RDS endpoint |
+| gigadb_db_user | gigadb |
+| gigadb_db_password | Pick a password |
+| gigadb_db_database | gigadb |
+| fuw_db_host | keep empty, it will be overwritten by the provisioning script with RDS endpoint |
+| fuw_db_user | fuw |
+| fuw_db_password | Pick a password |
+| fuw_db_database | fuw |
+| REVIEW_DB_DATABASE | reviewdb |
+| REVIEW_DB_PASSWORD | Pick a password |
+| REVIEW_DB_PORT | 5432 |
+| REVIEW_DB_USERNAME | reviewdb |
+| REVIEW_DB_HOST | reviewdb |
+| PORTAINER_PASSWORD | Pick a password |
+| remote_fileserver_hostname | files.yoursubdomain.gigadb.host | 
+| REMOTE_PUBLIC_HTTP_PORT | 80 |
+| REMOTE_PUBLIC_HTTPS_PORT | 443 |
```

Comment:
Can delete the `REMOTE_PUBLIC_HTTP_PORT` and `REMOTE_PUBLIC_HTTPS_PORT` rows here because they have already been defined above on lines 335 and 336.